src/cluster.c

```diff
@@ -4123,11 +4146,15 @@ sds clusterGenNodeDescription(clusterNode *node) {
     sds ci;
 
     /* Node coordinates */
-    ci = sdscatprintf(sdsempty(),"%.40s %s:%d@%d ",
+    ci = sdscatprintf(sdsempty(),"%.40s %s:%d@%d",
```

Comment:
if we need to add hostname support, we would need to bring in extensions mechanism to older version as well.